client/state/login/selectors.js

```diff
@@ -241,7 +234,7 @@ export const getCreateSocialAccountError = ( state ) =>
  * @returns {?object}         Error for the get social account request.
  */
 export const getRequestSocialAccountError = ( state ) =>
-	get( state, 'login.socialAccount.requestError', null );
+	state.login.socialAccount?.requestError ?? null;
```

Comment:
For `bearerToken`, `createError` and `requestError`, the optional chaining is redundant and the fallback is OK.

Reply:
Done.